Gemfile.lock

```diff
@@ -289,7 +289,7 @@ GEM
       railties (>= 5.0)
     rexml (3.2.5)
     ri_cal (0.8.8)
-    rmagick (4.3.0)
+    rmagick (5.0.0)
```

Comment:
This won't work without upgrading barcodes as that's still dependent on `RMagick` which is deprecrated in rmagic 5